models/app.model.js

```diff
@@ -60,3 +60,27 @@ exports.getAllArticles = () => {
     return result.rows;
   });
 };
+
+exports.getAllComments = (articleId) => {
+  const query = `
+        SELECT 
+            comment_id,
+            votes,
+            created_at,
+            author,
+            body,
+            article_id
+        FROM 
+            comments
+        WHERE 
+            article_id = $1
+        ORDER BY 
+            created_at DESC;
+    `;
+  return db.query(query, [articleId]).then((result) => {
+    if (result.rowCount === 0) {
+      return result.rowCount;
+    }
```

Comment:
This if statement can be refactored to avoid the need for the logic n the controller. If the rowCount is 0, you can just Promise reject here